frontend/src/main/pages/LeaderboardPage.js

```diff
@@ -51,6 +59,13 @@ export default function LeaderboardPage() {
     <div data-testid={"LeaderboardPage-main-div"} style={{backgroundSize: 'cover', backgroundImage: `url(${Background})`}}>
         <BasicLayout>
             <div className="pt-2">
+                <Button
+                  variant="danger"
```

Comment:
I wonder whether `primary` would be a better choice that `danger` here?   Our convention in the UI so far has generally been to use `danger` only for actions that delete something.    `primary` or `secondary` seem like better choices.  What do you think?

The choices are listed here: https://react-bootstrap.netlify.app/docs/components/buttons/ 

```suggestion
                  variant="primary"
```